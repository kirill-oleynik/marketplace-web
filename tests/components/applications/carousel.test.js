const React = require('react');
const { shallow } = require('enzyme');
const toJSON = require('enzyme-to-json').default;
const ApplicationsCarousel = require('../../../app/components/applications/carousel').default;

describe('#render', () => {
  test('it renders correctly', () => {
    const component = shallow(
      <ApplicationsCarousel
        applications={[{
          id: 1,
          title: 'Test1',
          description: 'Hello World!',
          logo: 'https://example.com/image.png'
        }]}
      />
    );

    expect(
      toJSON(component)
    ).toMatchSnapshot();
  });
});
