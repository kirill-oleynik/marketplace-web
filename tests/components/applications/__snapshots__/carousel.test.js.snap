// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#render it renders correctly 1`] = `
<Slider
  centerMode={true}
  dots={false}
  infinite={true}
  nextArrow={
    <Arrow
      direction="right"
      onClick={[Function]}
    />
  }
  prevArrow={
    <Arrow
      direction="left"
      onClick={[Function]}
    />
  }
  responsive={
    Array [
      Object {
        "breakpoint": 1200,
        "settings": Object {
          "slidesToShow": 3,
        },
      },
      Object {
        "breakpoint": 900,
        "settings": Object {
          "slidesToShow": 2,
        },
      },
      Object {
        "breakpoint": 640,
        "settings": Object {
          "slidesToShow": 1,
        },
      },
    ]
  }
  slidesToScroll={1}
  slidesToShow={4}
  speed={500}
>
  <ApplicationSlide
    application={
      Object {
        "description": "Hello World!",
        "id": 1,
        "logo": "https://example.com/image.png",
        "title": "Test1",
      }
    }
    className=""
  />
</Slider>
`;
